sympy/sets/fancysets.py

```diff
@@ -483,6 +492,127 @@ def _intersect(self, other):
             else:
                 return
 
+    def _union(self, other):
+        """
+
+        This function should only be used internally.
+        See Set._union for docstring
+
+        Used within the :class:`Union` class
+
+        Returns
+        =======
+        self._union(other) returns a new, joined ImageSet if self knows how
+        to join itself with other, otherwise it returns ``None``.
+
+        Examples
```

Comment:
Using docstrings is a good practice, I don't see how using them for internal functions should be a problem, also redundancy is not an issue as long as it is adding clarity to the explanation.